components/Nav/Nav.tsx

```diff
@@ -35,6 +35,7 @@ export const Nav = () => {
         'h-16 items-center border-b border-zinc-100 from-[#4E160E] to-zinc-950 px-4 dark:border-zinc-800 dark:bg-gradient-to-b',
         pathname === '/login' || pathname == '/register' ? 'hidden' : 'flex',
```

Comment:
q: Why is `flex` an option? Shouldn't it be default and hidden class just hides the component?

Reply:
That's a good point. I didn't think about that, but that makes more sense. It should also have a default styling of `flex` and it should either be hidden / not hidden based on the route. I will update it. Thanks, Shashi!